### PR TITLE
Updates to Virtual pin model

### PR DESCRIPTION
**Pull request by aaronburg**
Removed restrictions on what users can enter for virtual pin attribute :type.

Added web documentation on virtual pin description and usage

### CODE REVIEW

File: templates/web/guides/models/pins.md.erb [hunk 1]
```diff
@@ -222,6 +222,39 @@ ground_pin_groups.each do |id, group|
...
+  *  Tester utility pins that control relay states on the device interface board (DIB)
+  *  Tester channels to control components on the DIB
+
+~~~ruby
+add_virtual_pin(:virtual1, type: :utility_pin)
+add_virtual_pin(:virtual2, type: :utility_pin)
+add_virtual_pin(:virtual3, type: :relay_pin)
+
+add_virtual_pin_group :utility, :virtual1, :virtual2
+add_virtual_pin_group :relay, :relay_pin
```
Comment (ginty) [inline templates/web/guides/models/pins.md.erb]: I think this should say virtual3?

**Comment by ginty** [comment]
@priyavadan, assuming @aaronburg makes a quick patch for this, could you release a new version later today?
@welguisz's latest PR is already merged.

**Comment by priyavadan** [comment]
@ginty , sure can do!

File: templates/web/guides/models/pins.md.erb (hunk 1 above)
Comment (aaronburg) [inline templates/web/guides/models/pins.md.erb]: It certainly should.  Thanks for catching this.

--Aaron

**Comment by aaronburg** [comment]
Correction is made.

**Comment by priyavadan** [comment]
@aaronburg thanks!

@welguisz I'll make a new release just after lunch today.

**Comment by ginty** [comment]
@chrisnappi is also working on some stuff right now to be included

**Comment by priyavadan** [comment]
@ginty thanks for the info.

@chrisnappi please let me know when your stuff gets done and I will make a release; I am probably not going to be able to get to it till after lunch today anyways. Thanks!

**Comment by xnappo** [comment]
Okay, just need to add a test :)

**Pull request merged**